migrations/2024-10-28-125949_change_currency_type_in_dispute/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE dispute ALTER COLUMN currency TYPE "Currency" USING currency::"Currency";
```

Comment:
This query would fail, if there's any invalid data. We would need to run a migration on the sandbox and production environments to backfill the currency field in the dispute table, ensuring everything is in uppercase.

Reply:
I have added the migration query in the description

Reply:
Please ensure that this up.sql should be run only after deployment both in sandbox and production (along with the required migration to convert the existing data to upper case)